### ISSUE TITLE
Mike's Pod

### ISSUE THREAD

**Issue by romebell** [SEI-ATL/UNIT_2#22]
### Add your link to the ISSUE that you create here. Be sure to assign your Pod Leader

**Comment by JJURIZ** [comment]
How to incorporate Spotify Authorization into my app.

**Comment by tylerdance** [comment]
I'm trying to render the api data on my page but am getting `alerts is not defined` when I try to render my `finalObject` on line 47

<img width="1472" alt="Screen Shot 2020-11-30 at 3 19 43 PM" src="https://user-images.githubusercontent.com/72777766/100660216-b181ab00-331f-11eb-824d-8f7895e9496b.png">

![Screen Shot 2020-11-30 at 3 13 06 PM](https://user-images.githubusercontent.com/72777766/100660231-b6465f00-331f-11eb-9120-f1ab38c319d0.png)

**Comment by mkzarra** [comment]
@tylerdance Where is `alerts` coming from and what is it for?

**Comment by tylerdance** [comment]
Display "Saved Trails" page bug. Trying to just get the pages `<h1>` to load upon clicking the "Saved Trails" button on the homepage before attempting to actually save trails to database and display on page.

<img width="599" alt="Screen Shot 2020-12-01 at 10 11 07 AM" src="https://user-images.githubusercontent.com/72777766/100758785-e6dcd600-33bd-11eb-9028-b9ed1f5c7019.png">

Also have some questions about my post route and authentication. Thanks.

**Comment by mkzarra** [comment]
> Display "Saved Trails" page bug. Trying to just get the pages `<h1>` to load upon clicking the "Saved Trails" button on the homepage before attempting to actually save trails to database and display on page.
> 
> <img alt="Screen Shot 2020-12-01 at 10 11 07 AM" width="599" src="https://user-images.githubusercontent.com/72777766/100758785-e6dcd600-33bd-11eb-9028-b9ed1f5c7019.png">
> 
> Also have some questions about my post route and authentication. Thanks.

what behavior are you seeing when you click "Saved Trails"?

**Comment by tylerdance** [comment]
`Cannot GET /savedTrails`

**Comment by CoreyWilson319** [comment]
https://github.com/CoreyWilson319/Project-2/issues/1

**Comment by JJURIZ** [comment]
Inconsistent application of CSS rules in app. 
In this route you can see the style applied: 
![StyleApplied](https://user-images.githubusercontent.com/52257926/100891665-82864900-3487-11eb-8949-0d11987c5426.png)
After searching and getting a new route, style no longer applied:
![RouteBroken](https://user-images.githubusercontent.com/52257926/100891876-bbbeb900-3487-11eb-8033-a3075aeb0aa9.png)
Receiving the MIME error:
![MIME](https://user-images.githubusercontent.com/52257926/100891961-d3963d00-3487-11eb-8937-7d3969df1807.png)

Why is it working on some pages but not on others?

**Comment by tylerdance** [comment]
Saving trail to database bug

Post route:
<img width="796" alt="post route" src="https://user-images.githubusercontent.com/72777766/100909532-dcdcd500-349a-11eb-9421-d7a14be991fd.png">

`index.ejs` with "Save trail" button
<img width="1098" alt="save trail button" src="https://user-images.githubusercontent.com/72777766/100909649-00078480-349b-11eb-8b0e-b69315909f12.png">

Error message:
<img width="815" alt="error" src="https://user-images.githubusercontent.com/72777766/100909609-f3832c00-349a-11eb-83e8-31890855e86d.png">

**Comment by NikkiHmltn** [comment]
<img width="437" alt="Screen Shot 2020-12-02 at 9 52 10 AM" src="https://user-images.githubusercontent.com/65046641/100911555-400f3d00-3484-11eb-98e2-14cc85cb186e.png">
![Screen Shot 2020-12-02 at 9 52 19 AM](https://user-images.githubusercontent.com/65046641/100911566-41d90080-3484-11eb-9a9a-0c0138a4f1a0.png)

I'm having issues carrying the context over from the POST route. It's saying it's not defined. Am I allowed to bring context from a POST route? Also sidenote: I did change the route methods after I took the screenshot, and its not the cause of the issue.
